src/telegram.js

```diff
@@ -20,9 +20,10 @@ const pump = require('pump');
 const deprecate = require('depd')('node-telegram-bot-api');
 
 const _messageTypes = [
-  'text', 'audio', 'document', 'photo', 'sticker', 'video', 'voice', 'contact',
-  'location', 'new_chat_participant', 'left_chat_participant', 'new_chat_title',
-  'new_chat_photo', 'delete_chat_photo', 'group_chat_created'
+  'text', 'audio', 'document', 'game', 'photo', 'sticker', 'video', 'voice', 'video_note', 'contact',
```

Comment:
Let's just arrange this alphabetically, for less cognitive load for a user looking through source code.